docs/installation.md

```diff
@@ -391,65 +396,73 @@ To run opentelemetry collector as Systemd Service please apply following steps:
    /usr/local/bin/otelcol-sumo --version
    ```
 
-1. Create configuration file and save it as `/etc/otelcol-sumo/config.yaml`.
+1. Create configuration:
+
+   - Get [recommended configuration](https://github.com/SumoLogic/sumologic-otel-collector/blob/v0.57.2-sumo-1/examples/sumologic.yaml) from Sumo Logic repository and save it as `/etc/otelcol-sumo/sumologic.yaml`.
+   - Create your custom configuration in `/etc/otelcol-sumo/conf.d`, e.g. `/etc/otelcol-sumo/conf.d/common.yaml`
 
    > **IMPORTANT NOTE**:
    > It is recommended to limit access to the configuration file as it contains sensitive information.
    > You can change access permissions to the configuration file using:
    >
    > ```bash
-   > chmod 640 config.yaml
+   > mkdir -p /etc/otelcol-sumo/{conf.d,env}
+   > chmod 551 /etc/otelcol-sumo /etc/otelcol-sumo/{conf.d,env}
+   > chmod 440 /etc/otelcol-sumo/conf.d/common.yaml /etc/otelcol-sumo/sumologic.yaml
    > ```
 
 1. Create `user` and `group` to run opentelemetry by:
 
    ```bash
-   sudo useradd -mrUs /bin/false otelcol-sumo
+   sudo useradd -mrUs /bin/false -d /var/lib/otelcol-sumo otelcol-sumo
    ```
 
    > **IMPORTANT NOTE**:
    > This command will create a home directory for the user. By default, the `sumologic` extension stores the credentials in a subdirectory of the home directory.
    > However, if the user with name `otelcol-sumo` already exists, it won't be overwritten, so you should make sure that a home directory has been created for this user.
    >
-   > If you don't want the user to have a home directory, you should use `useradd` without the `m` flag
-   > and explicitely change the directory for saving the credentials, for example:
+   > If you don't want the user to have a home directory, you should use `useradd` with the `M` flag
+   > instead of `m` (`sudo useradd -MrUs ...`)
```

Comment:
```suggestion
   > If you don't want the user to have a home directory, you should use `useradd` with the `M` flag instead of `m` (`sudo useradd -MrUs ...`)
```